lib/terrafying/components/letsencrypt.rb

```diff
@@ -324,15 +291,54 @@ def renew(name, bucket, domains, options={})
                         "route53:ChangeResourceRecordSets",
                       ],
                       Resource:
-                        domains.map { | domain |
-                          "arn:aws:route53:::#{domain.zone.id[1..-1]}"
+                        @zones.reject { | z | z.empty? }.map { | zone |
```

Comment:
`(&:empty?)`

Reply:
using `nil?` as:
`NoMethodError: undefined method empty?' for #<Terrafying::Components::Zone:blah`